src/backend/executor/cypher_merge.c

```diff
@@ -57,33 +57,28 @@ const CustomExecMethods cypher_merge_exec_methods = {MERGE_SCAN_STATE_NAME,
                                                      exec_cypher_merge,
                                                      end_cypher_merge,
                                                      rescan_cypher_merge,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL,
-                                                     NULL};
+                                                     NULL, NULL, NULL, NULL,
+                                                     NULL, NULL, NULL, NULL};
 
 /*
- * Initializes the MERGE Execution Node at the begginning of the execution
+ * Initializes the MERGE Execution Node at the beginning of the execution
  * phase.
  */
 static void begin_cypher_merge(CustomScanState *node, EState *estate,
                                int eflags)
 {
     cypher_merge_custom_scan_state *css =
         (cypher_merge_custom_scan_state *)node;
-    ListCell *lc;
-    Plan *subplan;
+    ListCell *lc = NULL;
+    Plan *subplan = NULL;
 
     Assert(list_length(css->cs->custom_plans) == 1);
 
     // initialize the subplan
     subplan = linitial(css->cs->custom_plans);
     node->ss.ps.lefttree = ExecInitNode(subplan, estate, eflags);
 
+    /* TODO is this necessary? Removing it seems to not have an impact */
```

Comment:
Is this TODO completed as well?

Reply:
That is something that **needs** to be looked into. So, it is something pending. Could be for an intern?!

Reply:
Should we add this to the onboarding project?

Reply:
Sure